src/Sylius/Bundle/AdminBundle/Resources/config/grids/catalog_promotion.yaml

```diff
@@ -15,17 +15,17 @@ sylius_grid:
                     label: sylius.ui.name
                     sortable: ~
                 startDate:
-                    type: datetime
+                    type: twig
                     label: sylius.ui.start_date
                     sortable: ~
                     options:
-                        format: Y-m-d
```

Comment:
Cannot we just change the format to `Y-m-d H:i`?

Reply:
No because the goal is to move time below date. Problem solved.

Reply:
To make it clear: one-liner is fine by me, as long as it does not break display of the whole table (on lower resolutions for instance), @rafalswierczek told me that it works fine in one line also for lower resolutions so let's go with one line :)
It will be consistent with orders index for example :)